src/Preprocessor.zig

```diff
@@ -70,11 +70,7 @@ defines: DefineMap,
 tokens: Token.List = .{},
 generated: std.ArrayList(u8),
 token_buf: RawTokenList,
-char_buf: std.ArrayList(u8),
```

Comment:
this was only used during stringification, so I moved it there locally, is that okay?

Reply:
The reason it was in the `PreProcessor` was so that the buffer could be reused between stringifications redusing redundant allocations.